static/app/gettingStartedDocs/python/aiohttp.spec.tsx

```diff
@@ -26,7 +26,7 @@ describe('aiohttp onboarding docs', function () {
 
     // Does not render config option
     expect(
-      screen.queryByText(textWithMarkupMatcher(/tracesSampleRate: 1\.0,/))
```

Comment:
this one changed

Reply:
this was a typo from a previous pr i made🥲